extensions/participant-store-spi/src/main/java/org/eclipse/dataspaceconnector/registration/store/model/Participant.java

```diff
@@ -12,7 +12,7 @@
  *
  */
 
-package org.eclipse.dataspaceconnector.registration.api.model;
+package org.eclipse.dataspaceconnector.registration.store.model;
```

Comment:
I wonder if we should use the same type for api and store model. Should we maybe keep the Participant in the api packages and in the next PRs introduce a new `ParticipantRecord` containing more properties, eg. status as described here: https://github.com/Metaform/mvd/blob/main/registration-service/registration-service-tech-spec.md#3-participant-record?

Reply:
Let's do this modelling in the next PR as appropriate - for now there is no compelling reason not to use the DTO directly for persistence